.github/workflows/apt.yml

```diff
@@ -109,21 +109,22 @@ jobs:
         cd build
         # So high until-pass as a mitigation for https://github.com/robotology/gz-sim-yarp-plugins/issues/75
         ctest -T Test -T Coverage --repeat until-pass:20 --output-on-failure -C ${{ matrix.build_type }} . 
-      
-    - name: Generate a code coverage report
-      if: contains(matrix.build_type, 'Debug')
-      uses: threeal/gcovr-action@main
-      with:
-        excludes: |
-          build/*
-          install/*
-        # fail-under-line: 80
+
+    # Disabled due to https://github.com/robotology/gz-sim-yarp-plugins/issues/165
+    # - name: Generate a code coverage report
+    #   if: contains(matrix.build_type, 'Debug')
+    #   uses: threeal/gcovr-action@main
+    #   with:
+    #     excludes: |
+    #       build/*
+    #       install/*
+    #     # fail-under-line: 80
     
-    - name: Upload coverage reports to Codecov
-      if: contains(matrix.build_type, 'Debug')
-      uses: codecov/codecov-action@v4
-      env:
-        CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
+    # - name: Upload coverage reports to Codecov
+    #   if: contains(matrix.build_type, 'Debug')
+    #   uses: codecov/codecov-action@v4
+    #   env:
+    #     CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
I would delete these lines, if necessary we can find them in the git history.

Reply:
I remove code coverage related steps in https://github.com/robotology/gz-sim-yarp-plugins/pull/166/commits/1e5423653023c3c625875c05e2826e4b8e174b18